source/opt/decoration_manager.h

```diff
@@ -90,6 +94,8 @@ class DecorationManager {
   IdToDecorationInstsMap group_to_decoration_insts_;
   // The enclosing module.
   ir::Module* module_;
+  void RemoveInstructionFromTarget(ir::Instruction* inst,
```

Comment:
Need a method comment. 

Also, I prefer to see all methods before all data members.  Please move this up.

Reply:
done.